src/Paket/Commands.fs

```diff
@@ -625,6 +647,7 @@ with
         member this.Usage =
             match this with
             | Add _ -> "add a new dependency"
+            | Github _ -> "commands to manipulate gihub repository references"
```

Comment:
Could you please change `gihub` to `GitHub`?

Reply:
Absolutely! Opened #3100 for this.